apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/EvaluationEditor/Editor/Playground/Variables/index.tsx

```diff
@@ -0,0 +1,273 @@
+'use client'
+
+import { ReactNode, useEffect, useState } from 'react'
+
+import { Config, readMetadata } from '@latitude-data/compiler'
+import { ProviderLogDto } from '@latitude-data/core/browser'
+import {
+  formatContext,
+  formatConversation,
+} from '@latitude-data/core/services/providerLogs/formatForEvaluation'
+import {
+  Badge,
+  Button,
+  cn,
+  CodeBlock,
+  CollapsibleBox,
+  Icon,
+  Popover,
+  Text,
+  Tooltip,
+} from '@latitude-data/web-ui'
+import useDocumentLogWithMetadata from '$/stores/documentLogWithMetadata'
+
+const PARAMETERS = [
+  'messages',
+  'context',
+  'response',
+  'prompt',
+  'parameters',
+  'config',
+  'duration',
+  'cost',
+]
+
+const VariableSection = ({
+  title,
+  content,
+  tooltip,
+  height = '36',
+  popoverContent,
+}: {
+  title: string
+  content: string
+  tooltip?: string
+  height?: string
+  popoverContent?: ReactNode
+}) => (
+  <div className='flex flex-col gap-2'>
+    <div className='flex flex-row gap-2 items-center'>
+      <div className='flex flex-row gap-2 items-center'>
+        <Badge variant='accent'>{title}</Badge>
+        {tooltip && (
+          <Tooltip
+            trigger={
+              <div>
+                <Icon name='info' color='primary' />
+              </div>
+            }
+          >
+            {tooltip}
+          </Tooltip>
+        )}
+      </div>
+      {popoverContent && (
+        <Popover.Root>
+          <Popover.Trigger asChild>
+            <Button variant='nope'>
+              <Icon name='circleHelp' color='foregroundMuted' />
+            </Button>
+          </Popover.Trigger>
+          <Popover.Content className='bg-white shadow-lg rounded-lg p-4 max-w-xl'>
+            {popoverContent}
+          </Popover.Content>
+        </Popover.Root>
+      )}
+    </div>
+    <textarea
+      className={cn(
+        `w-full p-2 rounded-lg border bg-secondary resize-y text-xs`,
+        {
+          'h-36': height === '36',
+          'h-24': height === '24',
+        },
```

Comment:
If `height` only respond to these two values, type it with them instead of using just a string